railties/lib/rails/commands/server.rb

```diff
@@ -137,7 +137,9 @@ def log_to_stdout
         console.formatter = Rails.logger.formatter
         console.level = Rails.logger.level
 
-        Rails.logger.extend(ActiveSupport::Logger.broadcast(console))
+        unless ActiveSupport::Logger.logger_outputs_to?(Rails.logger, $stdout)
```

Comment:
I matched $stdout to the existing usage in the file. I'll switch them both over to STDOUT.